sklearn/calibration.py

```diff
@@ -117,13 +117,12 @@ class CalibratedClassifierCV(ClassifierMixin,
             ``cv`` default value if None changed from 3-fold to 5-fold.
 
     n_jobs : int, default=None
-        Number of jobs to run in parallel.
-        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
-        ``-1`` means using all processors.
-
-        Base estimator clones are fitted in parallel across cross-validation
-        iterations. Therefore parallelism happens only when cv != "prefit".
+        The number of jobs to run in parallel. For every cross-validation split
+        specified by ``cv``, a calibrated classifier is fitted using the cloned
```

Comment:
I _think_ the calibration happens as a postprocessing step IIUC, so here it'd be more like

```suggestion
        specified by ``cv``, a classifier is fitted using the cloned
```

Reply:
Good catch! Incorporating change